shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -498,7 +508,23 @@ impl CassandraConnection {
 
                 Self::process_cdrs_response(response)
             }
-            Self::Scylla { .. } => todo!(),
+            Self::Scylla { session, .. } => {
+                let statement = prepared_query.as_scylla();
+                let response = session.execute(statement, (value,)).await.unwrap();
+
+                if let Ok(rows) = response.rows() {
+                    rows.into_iter()
+                        .map(|row| {
+                            row.columns
+                                .into_iter()
+                                .map(|col| ResultValue::new_from_scylla(col.unwrap()))
```

Comment:
we need to handle null value instead of unwrap here, I suggest changing new_from_scylla to take `Option<Value>` so that we never forget to do the null handling?